dashboard/test/ui/features/acquisition_products/sign_up.feature

```diff
@@ -0,0 +1,22 @@
+@eyes
+Feature: Teacher can create a new account
+
+Scenario: Teacher can create an account with the new school association flow
+
+  Given I am on "http://studio.code.org/users/sign_up"
+  And I press keys "my@validemail.com" for element "#user_email"
+  And I press keys "password" for element "#user_password"
+  And I press keys "password" for element "#user_password_confirmation"
+  And I press "#signup_form_submit" using jQuery
+  # Add the experiment flag
+  And I am on "http://studio.code.org/users/sign_up?enableExperiments=school_association_v2"
```

Comment:
out of curiosity, why not include the experiment flag on line 6?

Reply:
It goes away when the page reloads (when the user moves to the finish sign up page), so I had to add it back here anyway!

Reply:
And there aren't any changes before that second page so no need to add it in the first one

Reply:
The behavior should stick around though right? But fair point that it's not needed earlier!

Reply:
Interestingly, this didn't persist. I tried moving it to the line above but it [reverts back to using the old school association flow](https://app.saucelabs.com/tests/ffbe10ebba9949e08624a19657544710#33). Going to update the PR to move it back to the second url!

Reply:
weird. Definitely no biggie -- thanks for humoring me on this!